stdlib/public/core/Character.swift

```diff
@@ -120,7 +120,7 @@ public struct Character :
   @warn_unused_result
   static func _smallSize(value: UInt64) -> Int {
     var mask: UInt64 = 0xFF
-    for var i = 0; i < 8; i += 1 {
+    for i in 0..<8 {
```

Comment:
Re: internal consistency, the C-style formatting is also spaced in the way I suggested (it's not `var i=0; i<8; i+=1`).

Comparing `0..<3` and `0 ..< 3`, I find the latter significantly easier to process in every case.
The decision ultimately doesn't affect my life in any way, but being an open forum, I wanted to put my vote in for readability.